src/reducers/profileReducer.js

```diff
@@ -15,9 +17,20 @@ const initialState = {
         barChart:{
 
          }
-    }
+    },
+    paginatedProfiles:[],
+    startIndex:0
 };
 
+const getAllButRejectedProfiles = profiles => {
+    return profiles.filter(profile => profile.status != "Rejected");
```

Comment:
Please strict check with !== (Not mandatory but best practice)

Reply:
Fixed